docs/src/app/components/pages/components/popover.jsx

```diff
@@ -0,0 +1,152 @@
+let React = require('react');
+let { Popover, RaisedButton, SelectField, TextField } = require('material-ui');
+let ComponentDoc = require('../../component-doc');
+let Code = require('popover-code');
+let CodeExample = require('../../code-example/code-example');
+
+
+let PopoverPage = React.createClass({
+  getInitialState() {
+    return {
+      selectValue:'1',
+      textValue:'here is a value',
+      activePopover:'none',
+    }
+  },
+
+  render() {
+
+    let componentInfo = [
+      {
+        name: 'Props',
+        infoArray: [
+          {
+            name: 'anchorOrigin',
```

Comment:
Looks like `open` is missing. By the way, can we call it `isOpen`, see this PR for more details https://github.com/callemall/material-ui/pull/1996.